### PR TITLE
ES translation of QuickFacts revised 2/Feb/19

### PR DESCRIPTION
**Pull request by BlackLotus64**
Updated according to the newer version @lh1008

### CODE REVIEW

**Comment by erciccione** [comment]
ping @lh1008

File: monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.md [hunk 1]
```diff
@@ -53,41 +54,41 @@ La palabra Monero es del lenguaje Esperanto. Los creadores decidieron usar Esper
...
 
 **La minería es accesible:** Cualquiera que pueda conectarse a un dispositivo o navegador web puede
 participar.
 
 ### IMPLICACIONES Y USOS EN LA VIDA REAL
 
-Dado que Monero es seguro, de comisiones bajas y sin fronteras, las personas pueden enviar dinero a pesar de estados o bancos corruptos y/o quebrados. Esto empodera económicamente al individuo de **[países opresores](https://www.reddit.com/r/Monero/comments/6wczty/how_monero_changed_my_life/)** o economías deprimidas.
+Dado que Monero es seguro, de comisiones bajas y sin fronteras, las personas pueden enviar dinero a pesar de estados o bancos corruptos y/o quebrados, y se pueden conducir negocios sin competidores husmeando información crítica. Esto empodera económicamente al individuo y negocios en **[países opresores](https://www.reddit.com/r/Monero/comments/6wczty/how_monero_changed_my_life/),** economías deprimidas o ambientes de negocio altamente competitivos.
```
Comment (lh1008) [inline monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.md]: "Critical information" can also be translated as "información sensible". Which one would you think could fit better the translation?

File: monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.md [hunk 2]
```diff
@@ -53,41 +54,41 @@ La palabra Monero es del lenguaje Esperanto. Los creadores decidieron usar Esper
...
-+ **Monero en circulación:** 16,570,126 XMR (Approximate)
-+ **Capitalización de mercado:** $1,503,928,296 USD (~0.77% capitalización del mercado de criptomonedas)
-+ **Actual recompensa por bloque:** 3.5984 XMR
+_(A partir del 1/Feb/2018)_
++ **Total de Nodos activos:** 1,564 (Source: https://monerohash.com/nodes-distribution.html)
++ **Tasa hash de la red:** 663.2 MH/s
++ **Promedio de Transacciones/Hora:** 149 (30-day average)
++ **CPU centrales protegiendo la red:** 22,234,297
++ **Monero en circulación:** 16,758,217 XMR (Approximate)
++ **Capitalización de mercado:** $743,944,945 USD (~0.77% capitalización del mercado de criptomonedas)
```
Comment (lh1008) [inline monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.md]: We did some changes on this line due to information gathered and noticed by @el00ruobuob in #99, take a look at the comments if you like. Let me add the string with the information we found. 

**Capitalización de mercado:** $732,932,071 USD (~0.65% capitalización del mercado de criptomonedas)

File: monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.md (hunk 1 above)
Comment (BlackLotus64) [reply monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.md]: I think it fits a bit better, I'll be changing it tomorrow as soon as I can.

File: monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.md (hunk 2 above)
Comment (BlackLotus64) [reply monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.md]: Got it, will change it in the next push.

File: monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.md (hunk 1 above)
Comment (lh1008) [reply monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.md]: ok great :).

**Comment by BlackLotus64** [comment]
Sorry for the long time, had a very hectic week, comments updated and rebased :)

File: monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.md [hunk 3]
```diff
@@ -121,8 +122,9 @@ A pesar de que Monero ya está disponible y está siendo usado a nivel global, l
 + **_[Análisis SWOT de Monero]()_**
 + **_[Conectarse con la Comunidad de Monero](https://getmonero.org/community/hangouts/)_**
```
Comment (lh1008) [inline monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.md]: This link is missing to be changed:

https://www.getmonero.org/es/community/hangouts/

**Comment by lh1008** [comment]
@BlackLotus64 pinging you so you can take a look at the suggested changes. Thanks.

File: monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.md [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
-Monero es una moneda digital segura, privada y no rastreable disponible para todos.
+Monero es una moneda digital segura, privada y no rastreable disponible para todos. Sus características lo hacen dinero fungible.
 
 # HOJA DE DATOS
 
-Los usuarios pueden gastar de una manera segura, nadie puede ver sus balances ni rastrear sus actividades.
+Los usuarios pueden gastar de una manera segura, nadie puede ver sus balances ni rastrear sus actividades a menos que el usuario les de el permiso.
 
-Revisado 11/14/2018 - [Versión para imprimir](http://www.monerooutreach.org/pubs/2018/QuickFacts/Quick-facts_es_version-para-imprimir.pdf)  
+Revisado 2/1/2019 - [Versión para imprimir](http://www.monerooutreach.org/pubs/2018/QuickFacts/Quick-facts_es_version-para-imprimir.pdf)
```
Comment (lh1008) [inline monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.md]: The date is not consistent with the other dates. 01/Feb/2019.

Comment (BlackLotus64) [reply monero-outreach-docs/translations/es/quickfacts/QuickFacts_es.md]: Got it :+1:

**Pull request merged**